morph_tool/__init__.py

```diff
@@ -1,9 +1,9 @@
 """Implementation of the morphology file converter."""
-from morphio import LogLevel, Morphology
+from morphio import LogLevel, Morphology  # noqa
```

Comment:
why the noqa?

Reply:
Because it is imported but not used

Reply:
right, sorry; I wonder why these are re-exported from morphio to morph_tool namespace; seems like a strange behavior.

Does the linting check the `__all__`, so we don't need that if they are added there?